Duplicati/Server/webroot/ngax/index.html

```diff
@@ -232,7 +232,7 @@
                 </ul>
             </div>
 
-            <div ng-view class="content">
+            <div ng-view class="content" style="width: 100%">
```

Comment:
Setting this attribute in the CSS file did not work by the way.

I could really use some help with css...